cmd/account/servicequotas/describe.go

```diff
@@ -63,24 +66,45 @@ func newDescribeOptions(streams genericclioptions.IOStreams, flags *genericcliop
 }
 
 func (o *describeOptions) complete(cmd *cobra.Command) error {
-	k8svalid, err := o.k8sclusterresourcefactory.ValidateIdentifiers()
+	k8svalid, err1 := o.k8sclusterresourcefactory.ValidateIdentifiers()
```

Comment:
Optionally, since you are using `err1` only within this scope of `if` statement, I would suggest to keep it limited in that, instead of making it available to the whole function:

```go
if k8svalid, err := o.k8sclusterresourcefactory.ValidateIdentifiers(); !k8svalid && err != nil {
		return err
	}
```